test_extensions/media_sharing/procs.h

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2019 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+#ifndef __PROCS_H__
```

Comment:
`__PROCS_H__` is a reserved identifier. See https://en.cppreference.com/w/c/language/identifier. Please use `#pragma once` or `MEDIA_SHARING_PROCS_H` for example.

Reply:
fixed

Reply:
You missed the point :). The problem was the double underscore followed by a capital letter...